pkg/api/handlers/libpod/generate.go

```diff
@@ -72,6 +73,7 @@ func GenerateSystemd(w http.ResponseWriter, r *http.Request) {
 		Wants:            query.Wants,
 		After:            query.After,
 		Requires:         query.Requires,
+		Envs:             query.Envs,
```

Comment:
Please also change `pkg/api/server/register_generate.go` for the swagger docs of the REST API